packages/contracts/package.json

```diff
@@ -32,6 +32,7 @@
         "@unirep/circuits": "2.0.0-beta-1",
         "@unirep/utils": "2.0.0-beta-1",
         "@zk-kit/incremental-merkle-tree.sol": "^1.2.0",
+        "big-integer": "^1.6.51",
```

Comment:
is it used in this package?

Reply:
It's a peer dependence of `solidity-coverage`. I guess it could be a peer dep